holidays/countries/ghana.py

```
@@ -101,3 +167,101 @@ class GH(Ghana):
+
+class GhanaIslamicHolidays(_CustomIslamicHolidays):
+    EID_AL_ADHA_DATES_CONFIRMED_YEARS = (2017, 2026)
+
+    EID_AL_FITR_DATES_CONFIRMED_YEARS = (2017, 2026)
+    EID_AL_FITR_DATES = {
+        2017: (JUN, 26),
+        2019: (JUN, 5),
+        2022: (MAY, 3),
+        2023: (APR, 22),
+        2024: (APR, 11),
+    }
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)